random.py

```diff
@@ -250,6 +250,38 @@ def generate_random_date(self):
     def run(self, view, **kwargs):
         self.insert(view, self.generate_random_date)
 
+class RandomTimeCommand(RandomText):
+
+    def generate_random_time(self):
+        hour = r.randint(0, 23)
+        minute = r.randint(0, 59)
+        second = r.randint(0, 59)
+        time = datetime.time(hour, minute, second)
+        return time.isoformat()
+
+    def run(self, view, **kwargs):
+        self.insert(view, self.generate_random_time)
+
+class RandomDatetimeRfc3339Command(RandomText):
+
+    def generate_random_datetime_rfc3339(self):
+        max_year = get_settings().get('max_year', (datetime.datetime.now().year,))[0]
```

Comment:
The default `max_year` and `min_year` seem a little random
Wouldn't it make sense to do any of these options:
- `datetime.datetime.now().year + 10` and `datetime.datetime.now().year - 10`? 
- `min_year = 1970` and `max_year = datetime.datetime.now().year + 30` 

I'm not convinced these are the best options

Reply:
Thank you and sure, both options look good. Choosing the first one.
(I was following your RandomDate pattern).